x-pack/plugins/enterprise_search/server/routes/app_search/crawler.test.ts

```diff
@@ -318,4 +318,51 @@ describe('crawler routes', () => {
       mockRouter.shouldThrow(request);
     });
   });
+
+  describe('POST /api/app_search/crawler/process_crawls', () => {
```

Comment:
Should this have said 

```suggestion
  describe('POST /api/app_search/engines/{name}/crawler/process_crawls', () => {
```

?